hooks/emitter/authz.go

```diff
@@ -43,15 +43,28 @@ func (h *Hook) handleMsgRevoke(msg *authz.MsgRevoke, detail common.JsDict) {
 	detail["url"] = msg.MsgTypeUrl
 }
 
-func (h *Hook) handleMsgExec(ctx sdk.Context, txHash []byte, msg *authz.MsgExec, detail common.JsDict) {
+func (h *Hook) handleMsgExec(
+	ctx sdk.Context,
+	txHash []byte,
+	msg *authz.MsgExec,
+	log sdk.ABCIMessageLog,
+	detail common.JsDict,
+) {
 	msgs, _ := msg.GetMessages()
 	grantee := msg.Grantee
 	for _, msg := range msgs {
 		switch msg := msg.(type) {
 		case *oracletypes.MsgReportData:
 			h.handleMsgReportDataFromGrantee(ctx, txHash, msg, grantee)
```

Comment:
we don't have to do it here too?

Reply:
yes. we don't want to show all oracle reports on the validator's account page (https://cosmoscan.io/account/band1r00x80djyu6wkxpceegmvn5w9nx65prgvpz48t#delegations) as it will have a lot of txs. 

also, we have validator page to show all reports already (https://cosmoscan.io/validator/bandvaloper1r00x80djyu6wkxpceegmvn5w9nx65prgqhxkzq#reports).